Microsoft.Toolkit.Uwp.UI.Controls/WrapPanel/WrapPanel.Data.cs

```diff
@@ -19,20 +20,20 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls.WrapPanel
     /// </summary>
     public partial class WrapPanel
     {
-        internal class UvMeasure
+        [System.Diagnostics.DebuggerDisplay("U = {U} V = {V}")]
+        private struct UvMeasure
         {
-            internal double U { get; set; }
+            private const double FACTOR = 10000;
 
-            internal double V { get; set; }
+            private double u, v;
+            internal double U { get { return u / FACTOR; } set { u = Math.Floor(value * FACTOR); } }
```

Comment:
Why? Doing this will cut all fractions, which is used in some controls. We should keep several number of digits.